### ISSUE TITLE
All Effects take forever to/cannot load

### ISSUE THREAD

**Issue by Galvarino** [rowak/nanoleaf-desktop#17]
More of a recent thing, and could be a problem caused by the amount of effects I have, but since there is a max capacity on the number of effects you can have that shouldn't be the reason. I don't get any error message for this, just infinite loading. One noteworthy thing is that it *does* recognize the current scene.
![image](https://user-images.githubusercontent.com/31280096/69906416-f1baf000-1390-11ea-856b-fcc0a5f59a12.png)
I've re-paired the panels many times to no avail. I'll update if I get it to work, since I haven't prodded at it much yet.

**Comment by rowak** [comment]
Interesting how it would suddenly just happen... Have you done a firmware update recently? This is the most likely cause, because I know Nanoleaf is changing parts of their api which might cause certain calls from the app to become outdated. Also, is it just the effects that don't work, or are there other UI elements (buttons, sliders, etc) that don't work or that give you an error? If you don't mind, could you post your console log when you run the app? It might help make the underlying problem more visible. I don't really have time to look into this right now myself, but I can suggest trying to clear the app's data file by clicking on the "Reset Settings" button from the menu in the top-left corner of the main window. Make sure to back up your settings before doing this if you don't want to lose them. Assuming that you are on Windows, this file can be found at "C:\\Users\\<YOUR_USER>\\AppData\\Roaming\\Nanoleaf for Desktop". It's unlikely that this will fix the problem, but it's worth a shot. I also always recommend this, but try running [v0.7.1](https://github.com/rowak/nanoleaf-desktop/releases/tag/v0.7.1) of the app to see if the effects load. Keep me updated.

**Comment by Galvarino** [comment]
Augh, so sorry for the incredibly late reply. A lot has happened concerning my nanoleaf aurora and remote since then. Essentially, I had to factory reset the device and it started working again afterwards. If you have a nanoleaf remote and you don't have a homekit setup, DO NOT try to update its firmware to 1.0.15. It will completely brick your remote and the only way to fix it is to download an older version of the nanoleaf app, force the remote to update back to an older version, then factory reset EVERYTHING and re-pair. It wasn't fun. Also (sorry for going so far off the topic of this discussion) a new desktop app was released for nanoleaf for windows which allows you to sync your screen, and funnily enough it has the same issue as yours with not being able to sync with some fullscreen apps, so they couldn't get it to work either. I still think this app reigns superior because it allows you to search the discovery page. Anyway, that is all, I will close the issue. To anyone with this issue going forward; try a soft reset, and if that doesn't work, go for the hard reset. Since it resets scenes data, it should definitely work. I never had a chance to test rowak's solution, but I would expect that to work too.
